### ISSUE TITLE
TypeError: Too few parameters for typing.Generator

### ISSUE THREAD

**Issue by mipapo** [flopp/GpxTrackPoster#15]
works until 2cf9558d6aaf6e02b477f62c1cb2039461b8edb1
breaks starting with 7842d4200fd7db9707953c8103c0e0c452ab22f7

```
(env) mpo@mpo-home:~/projects/flopp-GPX$ ./create_poster.py --type grid --gpx-dir "mytracks" --year 2017 --title "Running"Traceback (most recent call last):
  File "./create_poster.py", line 11, in <module>
    from src import track_loader
  File "/home/mpo/projects/flopp-GPX/src/track_loader.py", line 31, in <module>
    class TrackLoader:
  File "/home/mpo/projects/flopp-GPX/src/track_loader.py", line 156, in TrackLoader
    def __list_gpx_files(base_dir: str) -> typing.Generator[str]:
  File "/usr/lib/python3.6/typing.py", line 682, in inner
    return func(*args, **kwds)
  File "/usr/lib/python3.6/typing.py", line 1130, in __getitem__
    _check_generic(self, params)
  File "/usr/lib/python3.6/typing.py", line 662, in _check_generic
    ("many" if alen > elen else "few", repr(cls), alen, elen))
TypeError: Too few parameters for typing.Generator; actual 1, expected 3
```

**Comment by flopp** [comment]
fixed in 5697967c90351aad28421ef207fe4e5f6dbe3648

**Comment by mipapo** [comment]
Thanks! :+1:
